### ISSUE TITLE
**Bug report** updating padding on symbols

### ISSUE THREAD

**Issue by RaphyRaph** [DWilliames/paddy-sketch-plugin#51]
![bitmap](https://user-images.githubusercontent.com/3810802/36785823-29cc0b68-1c7c-11e8-8c6a-6f8a095495f4.png)

I'm seeing this after editing symbols in a file where I haven't set paddings anywhere.

**Comment by lukasoppermann** [comment]
Me too, all the time, it is pretty annoying. Maybe there could be an option to disable this feature?

**Comment by petebarr** [comment]
Yeah, I get the same thing. It pops up constantly and I have no idea why. Almost everytime I click on an element in a symbol that uses Paddy. e.g. just to change text

**Comment by highvoltaged** [comment]
Still got this after 1.0.6.
My symbol does not have any paddy but it still does it.

**Comment by highvoltaged** [comment]
It seems like it only affects old symbols, created with older paddy version. Testing further.

**Comment by DWilliames** [comment]
Yeah... I thought I fixed this in 1.0.6 – but it seems to not be working properly. I think I might remove the alert entirely; and not allow the options to automatically update al instances. I think that will solve a range of issues.

Sorry all; I'm sure that alert must be incredible annoying :/
